scripts/inject.mjs

```diff
@@ -50,6 +50,8 @@ async function init() {
         initOther('inject/mozilla.js', process.argv[2]);
     } else if (process.argv[2] == "apple") {
         initOther('inject/apple.js', process.argv[2]);
+    } else if (process.argv[2] == "android") {
+        initOther('inject/android.js', process.argv[2]);
```

Comment:
You'll need another change in https://github.com/duckduckgo/content-scope-scripts/blob/main/package.json#L9 to output this file and then check in the resulting built file also :)

Reply:
Yeah, I just realized this. Will add 👍